cms/templates/cms/home_page.html

```diff
@@ -91,24 +88,24 @@
              <img src="{% static 'images/online.png' %}" alt="">
           </div>
 
-          <h5 class="upper-case">Online Learning</h5>
-          <p>Take MIT graduate-level courses remotely, from anywhere in the world.</p>
+          <h5 class="upper-case">Convenient & Flexible</h5>
+          <p>Take online courses on a schedule that fits your needs, and meet motivated learners who share your interests.</p>
         </div>
         <div class="enroll-reason col-md-4">
           <div class="reasons-images-wrap">
             <img src="{% static 'images/blended.png' %}" alt="">
           </div>
 
-          <h5 class="upper-case">Blended Master’s</h5>
-          <p>Start a new, faster path to a Master’s degree from MIT or other universities.</p>
+          <h5 class="upper-case">Accelerate Your Master’s</h5>
+          <p>MicroMasters is a faster path to a master’s degree from MIT and other universities, and is recognized by employers for providing deep learning in specific career fields.</p>
```

Comment:
Should the "M" in "master's degree" be capitalized, or not? It's lowercase here, but capitalized in the `<h2>` element.